core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
+Item.RENAME.description=Renommer un job.
+ItemGroupMixIn.may_not_copy_as_it_contains_secrets_and_=Copie impossible de {0} car il contient des secrets et {1} possède {2}/{3} mais pas /{4}
+JDK.DisplayName=JDK
 Job.AllRecentBuildFailed=Tous les builds récents ont échoué.
-Job.BuildStability=Stabilité du build: {0}
+Job.BuildStability=Stabilité du build : {0}
 Job.NOfMFailed={0} des {1} derniers builds ont échoué.
 Job.NoRecentBuildFailed=Aucun build récent n''a échoué.
 Job.Pronoun=job
 Job.minutes=mn
-
-Queue.AllNodesOffline=Tous les noeuds avec le libellé ''{0}'' sont hors ligne
-Queue.BlockedBy=Bloqué par {0}
+Job.NoRenameWhileBuilding=Impossible de renommer un job en cours de construction.
+Job.you_must_use_the_save_button_if_you_wish=Vous devez utiliser le bouton Sauvegarder si vous voulez renommer un job.
+Label.GroupOf=groupe de {0}
+Label.InvalidLabel=libellé invalide
+Label.ProvisionedFrom=Provisionné depuis {0}
+LabelExpression.InvalidBooleanExpression=Expression booléenne invalide : {0}
+LabelExpression_ObsoleteMasterLabel=Cette expression contient le libellé <code>master</code> qui n''est plus utilisé pour le nœud principal. Utilisez <code>built-in</code> à la place. \
+  <a href="https://www.jenkins.io/redirect/built-in-node-migration/" rel="noopener noreferrer" target="_blank">Voir plus.</a>
+LabelExpression.LabelLink=<a href="{0}{2}">Label {1}</a> respecte {3,choice,0#no nodes|1#1 node|1<{3} nodes}{4,choice,0#|1# et 1 cloud|1< et {4} clouds}. \
+  Les droits ou autres restrictions apportées par des plugins peuvent réduire cette liste.
+LabelExpression.NoMatch=Aucun agent/cloud ne correspond à ce libellé.
+LabelExpression.NoMatch_DidYouMean=Aucun agent/cloud ne correspond à ce libellé. Vouliez-vous ‘{1}’ au lieu de ‘{0}’?
+ManageJenkinsAction.DisplayName=Administrer Jenkins
+MultiStageTimeSeries.EMPTY_STRING=
+ParametersDefinitionProperty.BuildButtonText=Build
+Queue.AllNodesOffline=Tous les nœuds avec le libellé ''{0}'' sont hors ligne
+Queue.LabelHasNoNodes=Aucun nœud avec le libellé ‘{0}’
+Queue.BlockedBy=Bloqué par ‘{0}‘
 Queue.HudsonIsAboutToShutDown=Jenkins est sur le point de se fermer
 Queue.InProgress=Un build est déjà en cours
 Queue.InQuietPeriod=En période d''attente. Expire dans {0}
-Queue.NodeOffline={0} est hors ligne
+Queue.NodeOffline=‘{0}‘ est hors ligne
 Queue.Unknown=???
-
+Queue.FinishedWaiting=Attente terminée
 Queue.WaitingForNextAvailableExecutor=En attente du prochain exécuteur disponible
-Queue.WaitingForNextAvailableExecutorOn=En attente du prochain exécuteur disponible sur {0}
+Queue.WaitingForNextAvailableExecutorOn=En attente du prochain exécuteur disponible sur ‘{0}‘
+Queue.init=Restoration de la file de builds
+Queue.node_has_been_removed_from_configuration={0} a été supprimé de la configuration
+Queue.executor_slot_already_in_use=Executor slot déjà utilisé
```

```suggestion
Queue.executor_slot_already_in_use=Emplacement d''exécuteur déjà utilisé
```
Not sure if this needs to be translated.